chapter_preliminaries/probability.md

```diff
@@ -1,304 +1,750 @@
-label, identifying objects in the image. We can also think of the label as another
-random variable. We can even think of all the metadata such as location, time, aperture, focal length, ISO, focus distance, and camera type as random variables.
-All of these are random variables that occur jointly. When we deal with multiple random variables, there are several quantities of interest.
 
-Given values $a$ and $b$, we may ask what the *joint probability* is that both values occur simultaneously, i.e. $P(A = a, B = b)$. This lets us determine, for instance whether $a$ implies $b$ or vice versa or whether they are related at all. Note that for any values $a$ and $b$ we have that $P(A=a, B=b) \leq P(A=a)$ and $P(A=a, B=b) \leq P(B = b)$. 
-This has to be the case, since for $A=a$ and $B=b$ to happen, $A=a$ has to happen *and* $B=b$ also has to happen. Thus, the intersection of $\{A=a\}$ and $\{B=b\}$ cannot be more likely than $\{A=a\}$ or $\{B=b\}$ individually.
 
-Due to the above inequality we know that $0 \leq \frac{P(A=a, B=b)}{P(A=a)} \leq 1$. We call this ratio the *conditional probability* and denote it by $P(B=b|A=a)$: it is the probability of $B=b$, provided that
-$A=a$ holds. Conditional probabilities behave just like unconditional ones, as long as we leave the conditioning unchanged. For instance, for disjoint events $\mathcal{B}$ and $\mathcal{B}'$ we have that 
-$P(\mathcal{B} \cup \mathcal{B}'|A = a) = P(\mathcal{B}|A = a) + P(\mathcal{B}'|A = a)$. Note that for convenience one typically writes $P(A, B)$ and $P(A|B)$ whenever the specific value is less important. 
+## Multiple Random Variables
 
-Using the definition of conditional probabilities, we can derive one of the most useful and celebrated equations in statistics: *Bayes' Rule*.
-By construction, we have that $P(A, B) = P(B|A) P(A)$ and $P(A, B) = P(A|B) P(B)$. Combining both equations yields 
+You might have noticed that we couldn't eve
+make it past the last section without
+making statements involving interactions 
+among multiple random variables
+(recall $P(X,Y) = P(X) P(Y)$).
+Most of machine learning 
+is concerned with such relationships.
+Here, the sample space would be 
+the population of interest,
+say customers who transact with a business,
+photographs on the internet,
+or proteins known to biologists.
+Each random variable would represent 
+the (unknown) value of a different attribute. 
+Whenever we sample an individual from the population,
+we observe a realization of each of the random variables.
+Because the values taken by random variables 
+correspond to subsets of the sample space 
+that could be overlapping, partially overlapping, 
+or entirely disjoint,
+knowing the value taken by one random variable
+can cause us to update our beliefs 
+about what values of another random variable are likely.
+If a patient walks into a hospital 
+and we observe that they 
+are having trouble breathing
+and have lost their sense of smell,
```

Comment:
Just to double-check: we use "they" and "their" refer to "a patient" (a singular) rather than he/she/his/her when the gender is unknown, correct?